config/locales/en.yml

```diff
@@ -182,13 +182,23 @@ en:
     local_fixity_check:
       type: 'Local Fixity'
       summary:
+        self:
+          in_progress: "Local fixity check is in progress this resource. If this doesn't resolve itself then contact DLS with a link to this resource via #digital-library."
```

Comment:
there's a word missing -- maybe it should be"in progress for this resource"

Reply:
Good catch - fixed.